sdks/java/testing/nexmark/src/main/java/org/apache/beam/sdk/nexmark/NexmarkConfiguration.java

```diff
@@ -42,6 +42,9 @@
   /** Where events come from. */
   @JsonProperty public NexmarkUtils.SourceType sourceType = NexmarkUtils.SourceType.DIRECT;
 
+  /** If provided, only generate events and write them to local file with this prefix. */
+  @JsonProperty public String generateInputFileOnlyPrefix = null;
```

Comment:
generateInputFilePathPrefix to be explicit?

Reply:
maybe `generateEventFilePathPrefix` ?

Reply:
SGTM

Reply:
Done